R/steps.R

```diff
@@ -101,11 +102,15 @@ Cicerone <- R6::R6Class(
 #' highlighted, generally a callback function. This is effectively a string that is evaluated JavaScript-side.
 #' @param on_next A JavaScript function to run when the next button is clicked (or its event triggered), 
 #' generally a callback function. This is effectively a string that is evaluated JavaScript-side.
+#' @param onNext alternative to above.
     step = function(el, title = NULL, description = NULL, position = NULL, 
       class = NULL, show_btns = NULL, close_btn_text = NULL,
       next_btn_text = NULL, prev_btn_text = NULL, tab = NULL, tab_id = NULL, is_id = TRUE,
-      on_highlighted = NULL, on_highlight_started = NULL, on_next = NULL) {
+      on_highlighted = NULL, on_highlight_started = NULL, on_next = NULL, element, onNext = NULL) {
```

Comment:
Why add `element` when we have `el`?
Why add `onNext` when we have `on_next`?

Reply:
The `el` is stored internally to the `R6` class as a list item entitled `element` - this is also how it is passed to driver.js. It's both for consistency with the driver.js API and because we're using an external source for the step content that is programmatically interfaced with the shiny app to generate the steps. Writing the steps from the `guide` `R6` `private` environment has them coming across with these names, so porting them back after modification would require reconfiguring the names to match the argument names for the R6 method.

It's mostly a convenience and doesn't really impede any functionality that I can see.

Reply:
I fail to see the convenience, it does the opposite if anything: it confuses.
Why add `onNext`when we have `on_next` that does the exact same thing?
Why add `element` when `el` does the exact same thing.

Reading the documentation it's not difficult to understand that `el` = `element` and `on_next` = `onNext`.

If you add `onNext` why not that for all other callbacks and arguments? 

How is the user supposed to understand this API? `$step("x", element = "y")`, you overwrite `el` with `element`, no way for the user to know that, unless we add needless checks that don't need to be there in the first place.

We need these removed,  they don't bring anything to the API but confusion